docs-devsite/vertexai-preview.functioncall.md

```diff
@@ -0,0 +1,42 @@
+Project: /docs/reference/js/_project.yaml
+Book: /docs/reference/_book.yaml
+page_type: reference
+
+{% comment %}
+DO NOT EDIT THIS FILE!
+This is generated by the JS SDK team, and any local changes will be
+overwritten. Changes should be made in the source code at
+https://github.com/firebase/firebase-js-sdk
+{% endcomment %}
+
+# FunctionCall interface
+A predicted \[FunctionCall\] returned from the model that contains a string representing the \[FunctionDeclaration.name\] and a structured JSON object containing the parameters and their values.
```

Comment:
Do FunctionCall and FunctionDeclaration.name have the proper syntax?

They look like this in the rendered docs: https://screenshot.googleplex.com/Xz3zxfw676YwTkJ.png